CHANGELOG.md

```diff
@@ -4,6 +4,353 @@ This changelog goes through the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.23.0-rc0](https://github.com/go-gitea/gitea/releases/tag/v1.23.0-rc0) - 2024-12-16
+
+* BREAKING
+  * Use camo.Always instead of camo.Allways (#32097)
+  * Remove SHA1 for support for ssh rsa signing (#31857)
+  * Use UTC as default timezone when schedule Actions cron tasks (#31742)
+  * Clear up old Actions logs (#31735)
+  * Fix OIDC introspection authentication (#31632)
+  * Rename project `board` -> `column` to make the UI less confusing (#30170)
```

Comment:
If it is a "breaking" change, it means that users need to do something or experience something.

But I do not see ``` Rename project `board` -> `column`  ``` is really breaking. What end users should do? Or is there anything really broken?

Reply:
Just the concept names changed. Many words in the UI changed from `board` -> `column`.

Reply:
I think users could understand it without "breaking"